src/index.js

```diff
@@ -7,12 +7,16 @@ import { serve, setup } from 'swagger-ui-express';
 import '@babel/polyfill';
 import swaggerSpecs from '../public/api-docs/swagger.json';
 import allRoutes from './routes/index';
+import customMessages from './utils/customMessages';
+import statusCodes from './utils/statusCodes';
 
 dotenv.config();
 
 const app = express();
-const port = process.env.PORT || 3000;
```

Comment:
Why did you remove the `|| 3000`? Could you put it back please?

Reply:
@the22mastermind this port number is supposed to be in `.env` file, when you add that value in `env` and you keep this 3000 here, it decreases the coverage. So that is why @karamuka  removed it I think.

Reply:
That is strange because on the previous PRs it didn't decrease the coverage.

Reply:
I agree with @de-scholar